tests/13-report.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+	<testsuite tests="0" failures="0" time="0.000" name="package/name">
```

Comment:
@jstemmer ,capturing the build output is problematic, since the build error is in the begginning of the test log. Another thing is that the build error is in the package level, and not in the testcase level.

An example for running test on 4 packages, which two of them failing will look like that:
```
# package/name/failing1
failing1/failing_test.go:15: undefined: x
# package/name/failing2
failing2/another_failing_test.go:20: undefined: y
=== RUN TestA
--- PASS: TestA (0.10 seconds)
PASS
ok  	package/name/passing1 0.005s
=== RUN TestB
--- PASS: TestB (0.10 seconds)
PASS
ok  	package/name/passing2 0.005s
FAIL	package/name/failing1 [build failed]
FAIL	package/name/failing2 [build failed]
```
For which I think that the xml should look like:
```
<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite tests="2" failures="0" time="0.100" name="package/name/passing1">
		<properties>
			<property name="go.version" value="1.0"></property>
		</properties>
		<testcase classname="name" name="TestA" time="0.100"></testcase>
	</testsuite>
	<testsuite tests="2" failures="0" time="0.100" name="package/name/passing2">
		<properties>
			<property name="go.version" value="1.0"></property>
		</properties>
		<testcase classname="name" name="TestB" time="0.100"></testcase>
	</testsuite>
	<testsuite tests="0" failures="1" time="0.000" name="package/name/failing1">
		<properties>
			<property name="go.version" value="1.0"></property>
		</properties>
	</testsuite>
	<testsuite tests="0" failures="1" time="0.000" name="package/name/failing2">
		<properties>
			<property name="go.version" value="1.0"></property>
		</properties>
	</testsuite>
</testsuites>
```
What do you think?